synfig-studio/src/gui/resources/ui/Makefile.am

```diff
@@ -0,0 +1,13 @@
+# $Id$
+
+MAINTAINERCLEANFILES = \
+	Makefile.in
```

Comment:
Why not write like the css Makefile.am style?
https://github.com/synfig/synfig/blob/520d9721a9d28243f8d2ba2efce4c4178a8ecf9a/synfig-studio/src/gui/resources/css/Makefile.am#L3

It's up to you but if you want to write in newline then use 2 spaces to indent

Reply:
Resolved.